doc/tutorials/using_sbl_on_up2.rst

```diff
@@ -89,51 +86,64 @@ An example of the configuration file ``uos.json``:
                              { "disk" : "clearlinux.img", "partition" : 2, "size" : "10G", "type" : "linux" } ],
-       "FilesystemTypes" : [ { "disk" : "clearlinux.img", "partition" : 1, "type" : "ext2" },
+       "FilesystemTypes" : [ { "disk" : "clearlinux.img", "partition" : 1, "type" : "vfat" },
                              { "disk" : "clearlinux.img", "partition" : 2, "type" : "ext4" } ],
        "PartitionMountPoints" : [ { "disk" : "clearlinux.img", "partition" : 1, "mount" : "/boot" },
            		          { "disk" : "clearlinux.img", "partition" : 2, "mount" : "/" } ],
-       "Version": 26880,
+       "Version": 27050,
        "Bundles": ["kernel-iot-lts2018", "openssh-server", "software-defined-cockpit", "os-core", "os-core-update"]
    }
 
 .. note::
    To generate the image with a specified version, please modify
    the "Version" argument, ``"Version": 26000`` instead
-   of ``"Version": 26880`` for example.
+   of ``"Version": 27050`` for example.
 
-Clone the source code of ``acrn-hypervisor`` and build SOS and LaaG image:
+Clone the source code of ``acrn-hypervisor`` for building:
 
 .. code-block:: none
+   
+   $ cd ~
+   $ git clone https://github.com/projectacrn/acrn-hypervisor
+
+Build SOS and LaaG image:
+   
+.. code-block:: none   
 
-   cd ~
-   git clone https://github.com/projectacrn/acrn-hypervisor
-   sudo ./acrn-hypervisor/devicemodel/samples/up2/create-up2-images.sh --images-type all \
-      --clearlinux-version 26880 --laag-json uos.json --acrn-code-path ~/acrn-hypervisor/
+   $ sudo -s 
+   # ./acrn-hypervisor/devicemodel/samples/up2/create-up2-images.sh --images-type all \
+      --clearlinux-version 27050 --laag-json uos.json --acrn-code-path ~/acrn-hypervisor/
 
+.. note::
+   Run ``create-up2-images.sh`` as root.
 
+.. note::
+   When building images, you can modify the ``--clearlinux-version`` argument 
+   to a specific version (such as 26800). To generate the images of SOS only, 
+   modify the ``--images-type`` argument to ``sos``.
+   
 This step will generate the images of SOS and LaaG:
 
 * sos_boot.img
 * sos_rootfs.img
 * up2_laag.img
 
-.. note::
-   When building images, you can modify the ``--clearlinux-version`` argument
-   to a specific version (such as 26800). To generate the images of SOS only,
-   modify the ``--images-type`` argument to ``sos``.
+Build the binary image ``partition_desc.bin`` for 
+GPT partitions, and change the partition layout 
+in ``partition_desc.ini`` if needed.
 
+.. code-block:: none
 
-We still need the binary image for GPT partitions and
-configuration file for flashing:
+   $ wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/gpt_ini2bin.py
+   $ wget https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/doc/tutorials/partition_desc.ini  
+   $ sudo -s
+   # python2 gpt_ini2bin.py partition_desc.ini>partition_desc.bin
```

Comment:
Do we still have python2 in Clear Linux?

Reply:
Python2 works in latest Clear Linux withn version 27050.

Reply:
Thanks for the confirmation. I believe the decision in the mid-term is to focus on Python3 so it would be good to plan to update this script to Python 3 before the Clear Linux team completely removes Python2 (this does not need to be done in this PR obviously).